packages/f/fxdiv/xmake.lua

```diff
@@ -9,11 +8,12 @@ package("fxdiv")
     add_versions("2020.12.09", "63058eff77e11aa15bf531df5dd34395ec3017c8")
 
     add_deps("cmake")
-    on_install("windows", "macosx", "linux", function (package)
+
+    on_install("windows", "macosx", "linux", function(package)
```

Comment:
please add space `function (`

Reply:
Why?

Reply:
Keeping the style consistent, you can see other packages

https://github.com/xmake-io/xmake-repo/blob/c53b49dbae4c330ae2cdf96ade2d63c08cec4150/packages/c/cairo/xmake.lua#L36

Reply:
They're not all consistent